HotelBooking/HotelBooking.DataAccessLayer/Entities/Reservation.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+public class Reservation
+{
+    //cena końcowa = cena za pokój * ilość nocy
```

Comment:
Czemu to zakomentowane ? :) Według mnie przydałyby się:
public DateTime DateFrom { get; set; }
public DateTime DateTo { get; set; }
public decimal Price { get; set; }

Reply:
u mnie nie ma komentarza :o